addons/lunch/data/mail_template_data.xml

```diff
@@ -35,6 +35,7 @@
                     <td valign="top" style="font-size: 13px;">
     <div>
         % set lines = ctx['lines']
+        % set sites = ctx['sites']
```

Comment:
Note that any change in jinja template completely break all translations: the whole content has to be extracted again and translated again. Is it necessary to do it in stable ? 14.3 is likely to be open to customers soon, so probably better target master.

Reply:
I know the downside of breaking translations but : 
- it is intended for our own prod (currently Happy Days doesn't know what needs to be delivered in GR1/GR2)
- no other customers are using 14.3 (yet) on saas, so it could be considered still as "master"

Reply:
Actually no it can not be considered as master. That's the whole purpose of a freezed version. Each added code can lead to bugs, and we have to update our code and templates manually as we are already on it. Anyway probably ok for this template in 14.3 .